base/siteorigin-widget.class.php

```diff
@@ -406,7 +406,7 @@ function add_defaults( $form, $instance, $level = 0 ){
 				$instance[$id] = $this->add_defaults( $field['fields'], $instance[$id], $level + 1 );
 			}
 			else if( $field['type'] == 'measurement' ) {
-				if( ! isset( $instance[$id] ) ) {
+				if( ! isset( $instance[$id] ) || $instance[$id] === false ) {
```

Comment:
Something's off if a measurement field can have it's value set to `false`. This might be treating the symptom, but there's probably an underlying issue that should be addressed too.